doc/source/user_guide/skimage2_migration.md

```diff
@@ -0,0 +1,30 @@
+(skimage2_migration)=
+
+# Transition to skimage2
+
+:::{hint}
+This document is a work in progress and still subject to change.
+:::
+
+scikit-image is preparing to release version 2.0 under the new namespace `skimage2`.
+This will affect both its import name and its package name on PyPI and elsewhere.
+Together with skimage2, we will release version 1.0.0 and 1.0.1 with the old API.
+Version 1.0.1 will emit a notification (FutureWarning) on import, telling users to either upgrade to skimage2 or pin to version 1.0.0.
+
+We do this to clean up long-standing issue without our API.
+Some of these issues involve changes to behavior that are difficult to address with conventional deprecations.
+Since we don't want to change behavior silently, we will introduce the new namespace which gives users an explicit way to upgrade to new behavior.
+
+You can find a more detailed description of our motivation and discussion leading up to this in {doc}`SKIP 4 <../skips/4-transition-to-v2>`.
```

Comment:
I'd like to keep this guide focused and concise and would rather link to motivation, etc elsewhere.
So should we update SKIP 4 or is SKIP 5 in order? :sweat_smile: I don't really see a reason not to update SKIP 4 which is still has the status "draft".

Thoughts?

Reply:
Probably worth doing a SKIP 5, since the idea evolved quite a bit over the different proposals.

Reply:
So should we publish SKIP 4, endorse it, and draft SKIP 5?

Reply:
Since it's still a draft, I'd say we are free to update and tweak further. I'd prefer that since having 3 SKIPs dealing with this problem might become confusing and would also duplicate a lot. 

For now I'd just keep in as a "draft". We are planning to tackle the breaking changes for skimage2 soon in a manner that allows toggling between old and new behavior (discussed a few days ago in a meeting). That way we can still make small changes to SKIP 4 if something comes up.

If we make significant changes we can address those in the "Discussion" or "Alternatives" sections. 

Does that sound reasonable?

Reply:
Yes, let's capture what we are currently doing accurately with as little effort as possible.